pages/gall/[id]/index.tsx

```diff
@@ -56,6 +57,8 @@ const Gall = ({ species, taxonomy, relatedGalls }: Props): JSX.Element => {
     species.hosts.sort((a, b) => a.name.localeCompare(b.name));
     const hostLinker = hostAsLink(species.hosts.length);
 
+    const notesSpeciesSource = (species.speciessource || []).find((s) => s.source?.title === "Gallformers ID Notes");
```

Comment:
It seems safer to look up by ID since the name can be edited by an admin.

I have not seen this before:
`(species.speciessource || []).find(...)`

wouldn't it be fucntionally equivalent and simpler to write it as:
`species.speciessource?.find(...)`
```suggestion
    const notesSpeciesSource = species.speciessource?.find((s) => s.source?.id === 58);
```

Reply:
Ahh yeah, I didn't realize the IDs would be consistent for all Notes records.

I think you're right about the optional chaining operator. The thing I did is old habit from before that operator was standard.